### ISSUE TITLE
Add project role email goes to owner not role recipient

### ISSUE THREAD

**Issue by camallen** [zooniverse/Panoptes#2044]
Linked to https://github.com/zooniverse/Panoptes-Front-End/issues/1749#issuecomment-260837490

The project owner gets the email about new roles and not the role recipient. [Here](https://github.com/zooniverse/Panoptes/blob/5b2194f5c88ec7778b3bc9510468932dd4a3550c/app/controllers/api/v1/project_roles_controller.rb#L14) should probably be using the linked user instead of the api user.

**Comment by camallen** [comment]
@zwolf you fancy fixing this bug ?

**Comment by mschwamb** [comment]
also interestingly when I added some people I didn't get the email at all. I added Chris to the project shortly after that as well and I didn't get an autoemail. Oddly only for the new user account both when I included and then revised the role. I wonder if there is also a bug where if your account is already on some other Zooniverse project then you don't get this email about be adding to a new different one
